### ISSUE TITLE
Accessibility of Accordion List

### ISSUE THREAD

**Issue by markaplet** [Office-of-Digital-Innovation/California-State-Web-Template#137]
The Accordion List (https://beta.template.webtools.ca.gov/sample/accordions.html#accordion) component is currently not accessible to screen readers. Upon activating the control neither Tab or down arrow will read the content. These are the most common keys that would presumably work. The control is announced as a tab (which is fine) so I first attempted to tab into the content assuming it was a manually controlled tab. See also https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html When that failed I tried using the down arrow key which should allow the screen reader to read the next line of text. The down arrow skipped right over the content and went to the next tab.

**Steps To Reproduce:**
1. Tab to the accordion control using a screen reader. I use NVDA.
2. Press enter or spacebar to open the accordion
3. Press either tab or down arrow in an attempt to read the content. 

**Expected results:** 
Using the Tab key should trow focus to the tab contents container like the example on the W3C Page: https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html  Since the content is directly below the accordion, AT users should be able to press the down arrow to read the next sentence.

**Comment by kkoryaka** [comment]
@markaplet, thank you for your feedbacks. and for bringing up great points. Recently, we have switched our template to Bootstrap 5 and jQuery-less approach, which means codebase for most of our components are now updated. Take a look again at our updated accordion list and see if your issue is still relevant. Our engineers are also taking thorough look at it to see if there are any potential accessibility issues.

**Comment by kkoryaka** [comment]
@markaplet our engineers tested the newest accordion list component using NVDA. They went through those steps that you have provided. Their have found that opening an accordion heading and pressing down arrow reads the content. Let me know if your findings are different.

**Comment by yesenia-gtz** [comment]
Closing as we are updating components for a11y compliance
